Examples/Scripts/FreeSurferPipelineBatch-long.sh

```diff
@@ -110,6 +111,9 @@ for i in ${!Subjects[@]}; do
       echo "About to use fsl_sub to queue ${HCPPIPEDIR}/FreeSurfer/LongitudinalFreeSurferPipeline.sh"
       queuing_command=("$FSLDIR/bin/fsl_sub" -q "$QUEUE")
   fi
+  else
+  	queuing_command=""
```

Comment:
If you use arrays, stay with arrays.
```suggestion
  	queuing_command=()
```

Reply:
ok